src/Scim/SimpleIdServer.Scim/SimpleIdServer.Scim.csproj

```diff
@@ -2,11 +2,13 @@
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
     <Description>SCIM2.0 implementation.</Description>
+	<LangVersion>latest</LangVersion>
```

Comment:
why is this needed?

Reply:
Because some changes in the pull request are not supported by previous c# version for example `??=`